mktestdocs/__main__.py

```diff
@@ -61,49 +109,39 @@ def check_docstring(obj, lang=""):
     """
     Given a function, test the contents of the docstring.
     """
+    executor = _executors[lang]
```

Comment:
What might be a good error message if the user supplies a language that isn't registered? I can imagine that a `KeyError` might be less friendly. I suppose `_executors` in `__main__.py` could also be a defaultdict that raises an error if the key is not filled. 

I'm curious what you think on this.

Reply:
I think that a `LookupError` is the right type, since it is a user-editable registry. But maybe a `ValueError` since `lang` is an argument. Either way I can write a better exception message for the traceback.

Reply:
`LookupError` indeed sounds good. The main thing is to make sure that we supply a meaningful error message that makes the user aware that the executor needs to be implemented.